modules/tactical/src/data_manager.ts

```diff
@@ -0,0 +1,67 @@
+/// <reference path="../../../typings/rx/rx.d.ts" />
+/// <reference path="../../../typings/rx/rx-lite.d.ts" />
+
+import {Observable, Subject} from 'rx';
+import {Backend, VersionedObject} from './backend';
```

Comment:
`VersionedObject` doesn't seem to be used.
Not sure if it should be used or removed from the imports.